stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/channel/list/ChannelListView.kt

```diff
@@ -117,12 +119,32 @@ public class ChannelListView @JvmOverloads constructor(
         scrollListener.setPaginationEnabled(enabled)
     }
 
-    public fun reachedEndOfChannels(endReached: Boolean) {
-        requireAdapter().endReached = endReached
+    public fun setChannels(channels: List<ChannelListItem>) {
+        requireAdapter().submitList(channels)
     }
 
-    public fun setChannels(channels: List<Channel>) {
-        requireAdapter().submitList(channels)
+    public fun showLoadingMore(show: Boolean) {
+        requireAdapter().let { adapter ->
+            val currentList = adapter.currentList
+            val loadingMore = currentList.contains(ChannelListItem.LoadingMoreItem)
+            val showLoadingMore = show && !loadingMore
+            val hideLoadingMore = !show && loadingMore
+
+            val updatedList = when {
+                showLoadingMore -> currentList.toMutableList().fluentAdd(ChannelListItem.LoadingMoreItem)
```

Comment:
This is the same as just

```suggestion
                showLoadingMore -> currentList + ChannelListItem.LoadingMoreItem
```

Reply:
Man, I write extensions and half the time I realize they're already available natively. This is one of the times I just... 🤦 .  Also GitHub is going to make you re-approve after I commit that suggestion.

Reply:
Almost everything is available in the Standard Library, that's one of the best things in the language 😄 

By the way, you'll also have to remove the extension and its import separately, unfortunately these one-line suggestions are rarely applicable on their own, as convenient as they are 😞